Identity/IdentityImplementing.swift

```diff
@@ -25,6 +25,22 @@ public class IdentityImplementing: Identity {
         self.graphQLManager = graphQLManager
     }
 
+    public func getSSO(provider: String, completion: @escaping (Result<String?, Error>) -> Void) {
+        graphQLManager.dispatch(
+            query: ApolloType.GetSsoQuery(provider: provider),
+            cachePolicy: .fetchIgnoringCacheData
```

Comment:
I would say that we could use the cache for this. I don't think it will change often, if ever.